src/uniprotkb/components/results/ResultsButtons.tsx

```diff
@@ -56,10 +56,20 @@ const ResultsButtons: FC<{
         </Suspense>
       )}
       <div className="button-group">
-        <button type="button" className="button tertiary disabled">
+        <button
+          type="button"
+          className={`button tertiary ${
+            selectedEntries.length === 1 ? '' : 'disabled'
```

Comment:
All of these should be using the button's native `disabled` attribute. I'd even go as far as saying this should be used **instead** of the class.